lib/download-node/index.js

```diff
@@ -55,7 +55,9 @@ async function main() {
           .on("entry", (header, stream, next) => {
             stream.on("end", next);
             if (header.name.endsWith("bin/node")) {
-              stream.pipe(createWriteStream(path.join(engineDir, "node")));
+              stream.pipe(
+                createWriteStream(path.join(engineDir, "node"), { mode: 0o777 })
```

Comment:
```suggestion
                createWriteStream(path.join(engineDir, "node"), { mode: 0o755 })
```
777 seems a bit overkill.